src/test/regress/expected/multi_create_table_triggers.out

```diff
@@ -0,0 +1,101 @@
+-- This tests file includes tests to show that we do not allow triggers
```

Comment:
minor: We nowadays  try to omit `multi_` prefix in the test names. Before  Citus become an extension, it was  useful to distinguish Postgres tests vs Citus tests in the same code-base, but now that's not useful anymore.